src/www/ui/api/Controllers/UploadController.php

```diff
@@ -1125,7 +1125,7 @@ public function getAllAgents($request, $response, $args)
       if (count($item['successfulAgents']) > 0) {
         $item['isAgentRunning'] = false;
       } else {
-        $item['currentAgentId'] = $agentDao->getCurrentAgentRef($item["agentName"])->getAgentId();
+        $item['currentAgentId'] = $agentDao->getCurrentAgentRef($item["agentName"])->getCurrentAgentId();
```

Comment:
Any reason to change this line?

Reply:
Yes, @dushimsam there is no property named "agentId" for  Agent entity. I think we can get currentAgentId instead of agentId since we are also assigning this value to $item['currentAgentId"].